docs/markdown/Contributions/releases/release-process.md

```diff
@@ -194,12 +194,10 @@ Note that this step can currently only be performed by a subset of maintainers d
 
 Go to the [documentation dashboard](https://dash.readme.com/). In the top left dropdown, where it says the current version, click "Manage versions". Click "Add new version" and use a "v" with the minor release number, e.g. "v2.9". Fork from the prior release. Mark this new version as public by clicking on "Is public?"
 
-Also, update the [Changelog](doc:changelog) page with the new release series at the top of the table. It's okay if there are no "highlights" yet.
-
```

Comment:
This won't work anymore because docs are from version control. Imo this shouldn't be part of the dev0 process. It should be part of the stable release process, along with the blog publishing